noir-projects/aztec-nr/authwit/src/cheatcodes.nr

```diff
@@ -17,7 +17,7 @@ where
     C: CallInterface<M>,
 {
     let target = call_interface.get_contract_address();
-    let inputs = cheatcodes::get_private_context_inputs(get_block_number());
+    let inputs = cheatcodes::get_private_context_inputs(get_block_number() - 1);
```

Comment:
Careful!

We used to do this because `this.blockNumber` in TXE means "the block I'm currently building" and we set the private context to "-1" so that we could do historical proofs. 

Will this change break that flow?